test/lint/lint-git-commit-check.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+#
+# Linter to check that commit messages have a new line before the body
+# or no body at all
+
+export LC_ALL=C
+
+EXIT_CODE=0
+
+while IFS= read -r commit_hash  || [[ -n "$commit_hash" ]]; do
+    n_line=0
+    while IFS= read -r line || [[ -n "$line" ]]; do
+        n_line=$((n_line+1))
+        length=${#line}
+        if [ $n_line -eq 2 ] && [ $length -ne 0 ]; then
+            echo "Your body is followed by a non-empty line. Subject lines should always be followed by a blank line."
+            EXIT_CODE=1
+            break
+        fi
+    done < <(git log --format=%B -n 1 "$commit_hash")
+done < <(git log origin/master..HEAD --format=%H)
```

Comment:
Could make sense to default this to `TRAVIS_COMMIT_RANGE` if available?

Reply:
fixed